### PR TITLE
MSP-3724: fix Supervisor entrypoint

### PR DESCRIPTION
**Pull request by Uburro**

### CODE REVIEW

File: images/worker/supervisord.sh
Comment (rdjjke) [inline images/worker/supervisord.sh:1]: You should also change the default supervisord config map in the Soperator code

Comment (Uburro) [reply images/worker/supervisord.sh:1]: https://github.com/nebius/soperator/pull/269/files#diff-7d981bdd8096c8bbdf86b9656bbd373645365ae39266a32f734c50af94138e73L210

what do you mean?

File: internal/render/worker/configmap.go
Comment (rdjjke) [inline internal/render/worker/configmap.go:1]:
```
res.AddLine("command=/opt/bin/slurm/slurmd_entrypoint.sh")
```
Here should be the slurmd binary

Comment (Uburro) [reply internal/render/worker/configmap.go:1]: https://github.com/nebius/soperator/pull/269/files#diff-9e90dcbf1100350ca32f5496062b222a6484fa33168142c891138e1d8619d4abL95

i do not think it's good idea

Comment (Uburro) [reply internal/render/worker/configmap.go:1]: maybe will we remove it `2>&1 | tee >(multilog s100000000 n5 /var/log/slurm/multilog)` ?

Comment (Uburro) [reply internal/render/worker/configmap.go:1]: if yes - it will be readable more

Comment (rdjjke) [reply internal/render/worker/configmap.go:1]: Ah, OK, I got how you made this.

> maybe will we remove it 2>&1 | tee >(multilog s100000000 n5 /var/log/slurm/multilog) ?
Hmm, I don't think we should.

So, now slurm_entrypoint.sh uses some env vars. Are you sure supervisord transfers them to the slurmd entrypoint?

Comment (rdjjke) [reply internal/render/worker/configmap.go:1]: I don't think it does at least for the `GRES` env var as it's not exported

File: images/worker/supervisord.sh
Comment (rdjjke) [inline images/worker/supervisord.sh:1]: Can we name it `supervisord_entrypoint.sh` to make it the same with all other containers?

File: internal/render/worker/configmap.go
Comment (Uburro) [reply internal/render/worker/configmap.go:1]: https://supervisord.org/configuration.html#environment-variables

Comment (Uburro) [reply internal/render/worker/configmap.go:1]: https://github.com/nebius/soperator/pull/269/commits/6dd4f8819755670c8a226792935650e6b1b79ea7
fixed

**Pull request merged**